### PR TITLE
ArduPlane: add 0 disable to FLTMODE_CH parameter doc

### PR DESCRIPTION
**Pull request by mirkix**
Change to the ArduCopter doc style and add 0 = disable to the doc (what I was missing)

### CODE REVIEW

File: ArduPlane/Parameters.cpp [hunk 1]
```diff
@@ -445,8 +445,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Param: FLTMODE_CH
     // @DisplayName: Flightmode channel
     // @Description: RC Channel to use for flight mode control
-    // @Range: 1 16
-    // @Increment: 1
+    // @Range: 0:Disabled,1:Channel 1,2:Channel 2,3:Channel 3,4:Channel 4,5:Channel 5,6:Channel 6,7:Channel 7,8:Channel 8,9:Channel 9,10:Channel 10,11:Channel 11,12:Channel 12,13:Channel 13,14:Channel 14,15:Channel 15,16:Channel 16
```
Comment (rmackay9) [inline ArduPlane/Parameters.cpp:448]: I think this should be in a new @Values line or the Range should be changed to "0 16".

**Review by rmackay9 — DISMISSED** [review]
see note above

**Comment by mirkix** [comment]
@rmackay9 Thanks Randy, changed it to @values .

**Comment by muramura** [comment]
If the setting of 0 is to take effect, the handling of the g.flight_mode_channel variable needs to be reviewed.
ex:
https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/control_modes.cpp#L142

**Comment by mirkix** [comment]
> If the setting of 0 is to take effect, the handling of the g.flight_mode_channel variable needs to be reviewed. ex: https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/control_modes.cpp#L142

But that will be handled by https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/control_modes.cpp#L143 already?

**Comment by muramura** [comment]
I am accessing memory beyond the array inside the get_radio_in method.
The argument type of this method is UINT8_T, so 0-1 is 255.
This 255 is used as an array index to refer to the wrong memory area.

https://github.com/ArduPilot/ardupilot/blob/master/libraries/RC_Channel/RC_Channel.h#L406

**Comment by WickedShell** [comment]
> I am accessing memory beyond the array inside the get_radio_in method. The argument type of this method is UINT8_T, so 0-1 is 255. This 255 is used as an array index to refer to the wrong memory area.
> 
> https://github.com/ArduPilot/ardupilot/blob/master/libraries/RC_Channel/RC_Channel.h#L406

That's not actually true here. The helper (at least on plane), will check the range and return nullptr ( https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/RC_Channel.h#L38 ) which then is safely handled here: https://github.com/ArduPilot/ardupilot/blob/master/libraries/RC_Channel/RC_Channel.h#L407

**Comment by mirkix** [comment]
> I wasted @mirkix san valuable time. I am sorry. I was wrong. I understand @WickedShell san point. Thank you, @WickedShell san.

@muramura Everything is fine, check it one more time does not hurt.

**Review by tridge — APPROVED** [review]

**Pull request merged**